webapp/sass/layout/_post.scss

```diff
@@ -377,6 +377,7 @@ body.ios {
         }
 
         .custom-textarea {
+            bottom: 0;
```

Comment:
The @mention isn't related to this I think. This is for the preview stuff that opens up after you click the preview link. And yes the @mention look fine.